server/routes/utils/voteModelUpdates.js

```diff
@@ -26,7 +26,7 @@ export async function registerVote(pollId, userId, option) {
             }
         );
 
-        return { status: 200, pollId: pollId, option: option };
+        return { pollId: pollId, option: option, newCounts: newCounts };
```

Comment:
You can write it as `return { pollId, option, newCounts }` with the same effect.